src/mudlet.cpp

```diff
@@ -2945,10 +2945,12 @@ void mudlet::setToolBarIconSize(const int s)
     if (mToolbarIconSize > 2) {
         mpMainToolBar->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
         mpButtonDiscord->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
+        mpButtonAbout->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
```

Comment:
`mpButtonAbout` doesn't exist here yet when you call it, that's why there's an issue.

Reply:
Oh snap, but there should be some kind of message when this happens, no?

Reply:
Hm... no, this is C++, not Lua. You are in a lot of control here, and when you screw up, the whole thing goes kaboom too :)